tests/sentry/tasks/test_post_process.py

```diff
@@ -173,19 +174,10 @@ def test_derive_invalid_platform(self, mock_derive_code_mappings):
         assert mock_derive_code_mappings.delay.call_count == 0
 
     @patch("sentry.tasks.derive_code_mappings.derive_code_mappings")
-    def test_derive_python(self, mock_derive_code_mappings):
-        data = {"platform": "python"}
-        self._call_post_process_group(data)
-        assert mock_derive_code_mappings.delay.call_count == 1
-        assert mock_derive_code_mappings.delay.called_with(self.project.id, data, False)
-
-    @patch("sentry.tasks.derive_code_mappings.derive_code_mappings")
-    def test_derive_js(self, mock_derive_code_mappings):
-        data = {"platform": "javascript"}
-        self._call_post_process_group(data)
-        assert mock_derive_code_mappings.delay.call_count == 1
-        # Because we only run on dry run mode even if the official flag is set
-        assert mock_derive_code_mappings.delay.called_with(self.project.id, data, True)
```

Comment:
I tried switching to `assert_called_with`, however, the `NodeData` object event though it was the same was not handling well comparing one of it's properties. IIRC correctly, a dictionary of properties would not compared to be the same because the key/values were out of order.

Instead of working to fix the comparison I decided to remove it. If we ever change the delay call, we should spend time to change it to not use a NodeData by extracting the information we need (e.g. stacktrace and project info).

Reply:
Removing the test seems okay. FYI Python introduced ordering in dictionaries, based on order of insertion as of 3.6 or 3.7.